monai/apps/pathology/transforms/post/array.py

```diff
@@ -320,3 +322,79 @@ def __call__(self, mask: NdarrayOrTensor, instance_border: NdarrayOrTensor) -> N
             marker = self.remove_small_objects(marker[None])
 
         return convert_to_dst_type(marker, mask, dtype=self.dtype)[0]
+
+
+class PostProcessHoVerNet(Transform):
+    def __init__(
+        self,
+        post_process_segmentation: Transform,
```

Comment:
I would suggest `seg_postpprocessing`, refer to:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/engines/trainer.py#L148
And seems the type can be `callable` instead of only `Transform`?

Thanks.

Reply:
Thanks for the suggestions, I have updated it in the latest commit.